crates/stackable-operator/src/commons/s3/crd.rs

```diff
@@ -97,14 +97,22 @@ pub enum S3AccessStyle {
     VirtualHosted,
 }
 
-/// Set a named AWS region, or defer to an auto-discovery mechanism.
-#[derive(Clone, Debug, Default, Deserialize, Eq, JsonSchema, PartialEq, Serialize)]
+/// Set a named AWS region.
```

Comment:
This could mean either AWS client _or_ bucket region

Reply:
In the context of S3, this will primarily mean the bucket region (the client region uses the same setting, but the bucket regions MUST be set correctly for valid sigv4 headers).

Reply:
Done in 6701b6b720408144b761269fed8eb1cd77ff30d8